components/admin/http-data-table/http-data-table.docs.mdx

```diff
@@ -0,0 +1,18 @@
+---
+description: A HttpDataTable component.
+---
+
+import { HttpDataTable } from './http-data-table';
+
+A component that does something special and renders text in a div.
+
+### Component usage
+```js
+<HttpDataTable>Hello world!</HttpDataTable>
```

Comment:
This is supposed to show how this component is used. Please update it appropriately